### ISSUE TITLE
About the train_fsns.py

### ISSUE THREAD

**Issue by feitiandemiaomi** [Bartzi/see#84]

**Comment by feitiandemiaomi** [comment]
@Bartzi Thanks for your good job
When I try to train the fsns according to readme, I find the loss are unusual, which shows as follows:
![image](https://user-images.githubusercontent.com/18325462/66990422-bb832300-f0f8-11e9-9c06-72fd2cbdb3d1.png)
ps: my train dataset is the original fsns test ,val dataset is the original val
I do not change code ,so it trains in default ,and I have two gusses,one is the my configuration of the fsns dataset has error ,the final gt  shows ：
![image](https://user-images.githubusercontent.com/18325462/66992725-dce60e00-f0fc-11e9-895c-fdf6a76c56a4.png)
Is it right? And the other guess is that the lr is too big, as the number of iterations increases, the lr becomes larger .
I will go on training.
Looking forward to your reply

**Comment by Bartzi** [comment]
Hmm looks like your starting learning rate is too high. My guess is you tried to train with a learning rate of `1e-3` you should set it to `1e-4`. It is totally normal that the printed learning rate increases this is because chainer calculates some crazy things for the Adam optimizer.

**Comment by feitiandemiaomi** [comment]
@Bartzi I will try, Thanks a lot!
